src/game_logic/turn.ts

```diff
@@ -114,13 +118,17 @@ export function resolveContracts (initialState: GameState): GameState {
  * been satisfied or, if none was found, -1.
  */
 function findIndexOfSatisfiedOpenContract (state: GameState): number {
-  return state.player.contracts.open.findIndex(contract => contract.isSatisfied(state))
+  return state.player.contracts.open.findIndex(contract => {
+    const satisfied = contract.isSatisfied(state)
+    return satisfied
+  })
```

Comment:
Why? lol

Reply:
for debugging. Needed an easy line to place a breakpoint. will revert